webui/react/src/types.ts

```diff
@@ -659,3 +661,15 @@ export interface PriorityScheduler {
   defaultPriority: number;
   preemption: boolean;
 }
+
+/* Jobs */
+
+export type Job = V1Job;
+export type JobType = Determinedjobv1Type;
+export type JobState = Determinedjobv1State;
+export type JobSummary = V1JobSummary;
```

Comment:
**question:** any ideas why some have the `Determined` prefix while others don't?

Reply:
https://github.com/determined-ai/determined/pull/3154#issuecomment-961354983